base/src/main/java/com/smartdevicelink/proxy/rpc/TireStatus.java

```diff
@@ -39,76 +39,123 @@
 import java.util.Hashtable;
 

```

Comment:
@kostyaBoss It looks like this whole file `TireStatus.java` was regenerated with the generator tool.  Could you just merge in the changes that are necessary for this PR?

Reply:
@santhanamk double-checked the TireStatus, there are only documentation updates, should I still revert?